apps/spruce/src/pages/adminSettings/tabs/ProvidersTab/ProvidersTab.tsx

```diff
@@ -0,0 +1,6 @@
+export const ProvidersTab = () => (
+  <div>
+    <h1>Providers</h1>
```

Comment:
H1, p should come from the Leafygreen Typography package i.e.

`import { Body, H1 } from "@leafygreen-ui/typography";`

and used in the code